examples/perturbation.py

```diff
@@ -98,29 +121,43 @@
 # This will determine how many crops are processed in parallel
 input_time_length = 450
 # final_conv_length determines the size of the receptive field of the ConvNet
-model = Deep4Net(in_chans=64, n_classes=2, input_time_length=input_time_length,
-                 filter_length_3=5, filter_length_4=5,
-                 pool_time_stride=2,
-                 stride_before_pool=True,
-                 final_conv_length=1)
+model = Deep4Net(
+    in_chans=64,
+    n_classes=2,
+    input_time_length=input_time_length,
+    filter_length_3=5,
+    filter_length_4=5,
+    pool_time_stride=2,
+    stride_before_pool=True,
+    final_conv_length=1,
+)
 if cuda:
     model.cuda()
 
 from torch.optim import AdamW
 import torch.nn.functional as F
 
-optimizer = AdamW(model.parameters(), lr=0.01, weight_decay=0.5 * 0.001)  # these are good values for the deep model
-model.compile(loss=F.nll_loss, optimizer=optimizer, iterator_seed=1, cropped=True)
+optimizer = AdamW(
+    model.parameters(), lr=0.01, weight_decay=0.5 * 0.001
+)  # these are good values for the deep model
+model.compile(
+    loss=F.nll_loss, optimizer=optimizer, iterator_seed=1, cropped=True
+)
 
 ##############################################################################
 # Run the training
 # ----------------
 
 input_time_length = 450
-model.fit(train_set.X, train_set.y, n_epochs=30, batch_size=64,
-          scheduler='cosine',
-          input_time_length=input_time_length,
-          validation_data=(valid_set.X, valid_set.y),)
+model.fit(
+    train_set.X,
+    train_set.y,
+    n_epochs=30,
+    batch_size=64,
+    scheduler="cosine",
+    input_time_length=input_time_length,
+    validation_data=(valid_set.X, valid_set.y),
```

Comment:
I personally don't like exploding like this all function calls it makes very long files. It's not part of the pep8 guidelines.